internal/http/services/owncloud/ocs/shares.go

```diff
@@ -651,7 +651,10 @@ func (h *SharesHandler) listSharedWithMe(r *http.Request) []*collaboration.Recei
 
 	lrs := collaboration.ListReceivedSharesRequest{}
 	// TODO(refs) handle error...
-	shares, _ := c.ListReceivedShares(r.Context(), &lrs)
+	shares, err := c.ListReceivedShares(r.Context(), &lrs)
+	if err != nil {
+		panic(err)
```

Comment:
do we really want to panic?

Reply:
Not really, just quick workaround. All that code needs propper error handling, you created that first mockup, can you followup on this PR and update it correctly?

Reply:
Will update all that with proper error handling, it's about time I do it :)

Reply:
@refs any update on this?